doc/admin-guide/logging/formatting.en.rst

```diff
@@ -451,6 +453,12 @@ cqcl  Client Request         Client request content length, in bytes.
 cqhl  Client Request         Client request header length, in bytes.
 cqql  Client Request         Client request header and content length combined,
                              in bytes.
+cqqtl Client Request         Same as cqql_, but for the first transaction on a
```

Comment:
We may want a note (maybe as a limitation) about the accuracy. In some cases, the logged value wouldn't really reflect the amount of data transferred on wire to make the first transaction.